src/common/components/private-repos-info/index.js

```diff
@@ -1,10 +1,11 @@
-import React, { Component } from 'react';
+import React, { Component, PropTypes } from 'react';
```

Comment:
We are not yet on React 15.5 that deprecates that, but that is something to consider sooner then later I guess.